internal/lightcone/abundance/cornucopia/cornucopia.go

```diff
@@ -0,0 +1,60 @@
+package cornucopia
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	CornucopiaBuff  key.Modifier = "cornucopia-buff"
+)
+
+func init() {
+	lightcone.Register(key.Cornucopia, lightcone.Config{
+		CreatePassive: Create,
+		Rarity:        3,
+		Path:          model.Path_ABUNDANCE,
+		Promotions:    promotions,
+	})
+	
+	modifier.Register(CornucopiaBuff, modifier.Config{
+		Listeners: modifier.Listeners{
+			OnBeforeAction: buffHealsOnSkillUlt,
+			OnAfterAction: removeHealBuff,
+		},
+		Stacking: modifier.Unique,
```

Comment:
nit: do not need to specify `Stacking` when using the default behavior (`Unique`)